src/pages/hcp/api-docs/vault-secrets/[[...page]].tsx

```diff
@@ -62,6 +62,15 @@ const GITHUB_SOURCE_DIRECTORY = {
 	ref: 'main',
 }
 
+/**
+ * Data to power the status page indicator in the header area.
+ */
+const STATUS_INDICATOR_CONFIG = {
+	pageUrl: 'https://status.hashicorp.com',
+	endpointUrl:
```

Comment:
For HCP services, I think we'll be able to consistently use `endpointUrl` and get the expected format back. This configuration may need to evolve over time when we onboard new products (for example if their status data is from some other API service), but I figure we'll cross such bridges when we get to them 👍